slices/libnss-mdns.yaml

```diff
@@ -0,0 +1,28 @@
+package: libnss-mdns
+
+essential:
+  - libnss-mdns_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libnss-mdns_mdns4
+      - libnss-mdns_mdns6
+    contents:
+      /usr/lib/*-linux-*/libnss_mdns.so.*:
+      /usr/lib/*-linux-*/libnss_mdns_minimal.so.*:
```

Comment:
How about adding a `mdns` slice of it's own and then `libs` can include all of them together?